client/src/components/reusable/Carousel.jsx

```diff
@@ -36,17 +36,17 @@ export function handleRightClick(ev) {
 
 export default function Carousel({ children }) {
   return (
-    <div style={{ display: 'flex' }}>
-      <button onClick={handleLeftClick} style={{ height: '20px', marginTop: 'auto' }}>left</button>
+    <div style={{ display: 'flex', justifyContent: 'center' }}>
+      <button type="button" onClick={handleLeftClick} style={{ height: '20px', marginTop: 'auto' }}>left</button>
```

Comment:
Just a note for us to put these styles into its own sass file later for reusability